### ISSUE TITLE
Artifacts with new trained models

### ISSUE THREAD

**Issue by Azur231** [TheLastBen/fast-stable-diffusion#1311]
I had already trained several models -- with pretty good results. 
![1](https://user-images.githubusercontent.com/122198447/211184954-4c81c52c-90a3-40b6-a557-96c40d4e952e.jpg)

However, I have now used the "new" Dreambooth with the new training method for the first time and get very unattractive results. Many of the trained faces and bodies show artifacts. 
![00179-3610259197-photo, beautiful Keanu Reeves, wearing Sweater, softlight, low light, fog, shallow dof, 2 6 mm,](https://user-images.githubusercontent.com/122198447/211184961-6904c358-f0ac-4e3b-a47c-619edad13eac.png)

I have already restarted the training several times, used different numbers of steps between 150 and 650. Am I doing something wrong? Or is it because of the new training method?

**Comment by micha42-dot** [comment]
Just came here because I have exactly the same problem. I wanted to remake a model that I had trained in december with higher resolution images...  832 instead of 512. And I just get fragmented faces or colorful mishmash out. Does the old Dreambooth version still exist? Or is the old training method still available? It took more steps and time, but I have already read on Reddit that the results of the old method were also better ... especially to train eyes.


![00066-732650475-emma watson  cyberpunk mercenary in a military vest ( blade runner 2 0 4 9, cyberpunk 2 0 7 7 )  orientalist portrait by john wi](https://user-images.githubusercontent.com/80447964/211185453-f41ed9fa-8220-48c4-b991-6999a4a7eacd.png)
![00168-213168421-photo, beautiful AriGrande, wearing Sweater, softlight, low light, fog, shallow dof, 2 6 mm,](https://user-images.githubusercontent.com/80447964/211185458-69052f93-378e-4326-a526-ae6660f67609.png)

![00063-500321489-emma watson  cyberpunk mercenary in a military vest ( blade runner 2 0 4 9, cyberpunk 2 0 7 7 )  orientalist portrait by john wi](https://user-images.githubusercontent.com/80447964/211185454-41341fab-69e1-4e79-a1e8-39163b97c509.png)

**Comment by TheLastBen** [comment]
If you want the old settings, just set the learning rate to 2e-6.
if you want to keep the current learning rate at 1e-5, use 10 instance images and don't go above 1000 steps for unet and keep the textenc below 250 steps.

Also, try with the V2-512 or 768, they produce impressive results after training.

**Comment by Azur231** [comment]
Yeah, just hopped to V2-768. But it really looks that the new and faster learing rate delivers less consistent results. The eyes in particular seem less well defined and have visible artefacts.

**Comment by TheLastBen** [comment]
reduce the learning rate and increase the unet steps

**Comment by Azur231** [comment]
Thx. I will try differend workflows. Right now it seems that 2e-6 delivers better eyes and a more defined skin.

**Comment by TheLastBen** [comment]
you can start with 1e-5 for 400 steps, to get a faster convergence, then you can resume for 1500 steps with 2e-6, there are a lot of ways to save time training.

**Comment by Azur231** [comment]
I will try that. But time is not my priority. I'm trying to get detail and consistency.

Can you tell me what the Text_Encoder_Learning_Rate in the old version was?

**Comment by TheLastBen** [comment]
same as the unet, 2e-6

**Comment by Azur231** [comment]
thx

**Comment by micha42-dot** [comment]
Thank you very much for the help. Maybe it's just my imagination, but I don't get as good results with the new training method as I did with the old Dreambooth version, even if I follow the instructions and your comments. 

Maybe I'm also a little overwhelmed with the settings and training rates. 

You say the old UNet_Learning_Rate was 2e-6, right? So I would have to calculate around 230 steps per Picture like in the old Dreambooth, right?

And how many text encoder training steps should I calculate per image? There was a default rate in the old Dreambooth, right?

Sorry if I'm being a pain, but I feel really stupid right now and a little frustrated because it just doesn't seem to work as well as it did before.

**Comment by TheLastBen** [comment]
the text encoder steps should not be set above 400-600, 400 is more than enough for the whole training.

**Comment by chewtoys** [comment]
December Version
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/b46128c74a31f2c329eb159b51e3235e53295ebe/fast-DreamBooth.ipynb

Images 14
UNet_Training_Steps 2000
2e-6
Text_Encoder_Training_Steps 300
2e-6

Boom done.
